From 5b38b4c3835b9f622b0798c2afce9d635c7f644e Mon Sep 17 00:00:00 2001
From: Dave Airlie <airlied@redhat.com>
Date: Fri, 4 Dec 2009 10:18:02 +1000
Patch-mainline: 2.6.33
References: fate#310916
Git-commit: 01d01ba947670cf58f22119fc126fdf39078f6ba
Subject: [PATCH 0025/2588] drm/mm: fixup typo in debug functions.

Free and used were reversed.

Signed-off-by: Dave Airlie <airlied@redhat.com>
(cherry picked from commit 01d01ba947670cf58f22119fc126fdf39078f6ba)

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 drivers/gpu/drm/drm_mm.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/gpu/drm/drm_mm.c b/drivers/gpu/drm/drm_mm.c
index 97dc5a4..1f0d717 100644
--- a/drivers/gpu/drm/drm_mm.c
+++ b/drivers/gpu/drm/drm_mm.c
@@ -395,7 +395,7 @@ int drm_mm_dump_table(struct seq_file *m, struct drm_mm *mm)
 		else
 			total_used += entry->size;
 	}
-	seq_printf(m, "total: %d, used %d free %d\n", total, total_free, total_used);
+	seq_printf(m, "total: %d, used %d free %d\n", total, total_used, total_free);
 	return 0;
 }
 EXPORT_SYMBOL(drm_mm_dump_table);
-- 
1.7.6

